### DISCUSSION TITLE
XYZ for testing XSS findings?

### DISCUSSION THREAD

**Discussion by SafeLanester** [GEOLYTIX/xyz#1371] [Q&A]
Hi , I hope this message reaches you in good health and spirits. I was wondering if you could explain how to use XYZ for testing False Negatives and False Positives. I want to use both Xsstrike and that if I can.

**Comment by dbauszus-glx** [comment]
@SafeLanester I am not sure I understand the use case.

**Comment by SafeLanester** [comment]
I apologize for not being clear, I seen on s0md3v/XSStrike in the FAQ section
" What's the false positive/negative rate?
There can be false positives while crawling because crawling skips thorough checks. If XSStrike marks a webpage as vulnerable while crawling, you should run a scan on that particular webpage for thorough scanning.

When you scan a single webpage, XSStrike makes use of a browser engine to ensure that the payload works and hence ensures zero false positives.

XSStrike already covers all the common + some special contexts but there can be false negatives if the injection requires some special strategy.

Tool xyz works against the target, while XSStrike doesn't!
Please use that other tool."
   Thats where I was wondering how xyz might work where XXStrike doesn't...
